cypress/integration/price/pricelist_with_currency_CHF_and_precision_3_spec.js

```diff
@@ -0,0 +1,29 @@
+import { PriceList } from '../../support/utils/pricelist';
+import { Pricesystem } from '../../support/utils/pricesystem';
+import { humanReadableNow } from '../../support/utils/utils';
+
+describe('Create Pricelist for Automatic End2End Tests with cypress https://github.com/metasfresh/metasfresh-e2e/issues/195', function() {
+    const date = humanReadableNow();
+    const priceSystemName = `TestPriceSystem ${date}`;
+    const priceListName = `TestPriceList ${date}`;
+    it('Create new Pricesystem', function() {
+        cy.fixture('price/pricesystem.json').then(priceSystemJson => {
+            Object.assign(new Pricesystem(), priceSystemJson)
+            .setName(priceSystemName)
+            .apply();
+        });
+    });
+    
+    it('Create new Pricelist with currency CHF and precision 3', function() {
+        cy.fixture('price/pricelist.json').then(pricelistJson => {
+            Object.assign(new PriceList(), pricelistJson)
+            .setName(priceListName)
+            .setPriceSystem(priceSystemName)
+            .setCountry(`Switzerland`)
```

Comment:
I thought about it, but since the whole point of the test is using those specific values, I thought it's not the case. Will change, tho

Reply:
When we'll start working on https://github.com/metasfresh/metasfresh-e2e/issues/286, having all the variables at the top of the script will really help since there will be no need to hunt for them inside the test.